File: adicionar_cidade.php
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $nome = $_POST["nome"];
    $temperatura = $_POST["temperatura"];
    $clima = $_POST["clima"];

    // Conectar ao banco de dados SQLite
    $db = new SQLite3('cidades.db');

    // Preparar a consulta SQL para inserir a cidade
    $query = $db->prepare("INSERT INTO cidades (nome, temperatura, clima) VALUES (:nome, :temperatura, :clima)");
    $query->bindParam(':nome', $nome, SQLITE3_TEXT);
    $query->bindParam(':temperatura', $temperatura, SQLITE3_FLOAT);
    $query->bindParam(':clima', $clima, SQLITE3_TEXT);

    // Executar a consulta SQL
    $result = $query->execute();

    if ($result) {
        echo "Cidade adicionada com sucesso!";
    } else {
        echo "Erro ao adicionar cidade!";
    }

    $db->close();
}
?>

File: adicionar_cidade.py
import sqlite3
from flask import Flask, request

app = Flask(__name__)

@app.route('/adicionar_cidade', methods=['POST'])
def adicionar_cidade():
    if request.method == 'POST':
        nome = request.form['nome']
        temperatura = request.form['temperatura']
        clima = request.form['clima']

        # Conectar ao banco de dados SQLite
        conn = sqlite3.connect('cidades.db')
        cursor = conn.cursor()

        # Inserir a cidade na tabela
        cursor.execute("INSERT INTO cidades (nome, temperatura, clima) VALUES (?, ?, ?)", (nome, temperatura, clima))
        conn.commit()
        conn.close()

        return 'Cidade adicionada com sucesso!'

if __name__ == '__main__':
    app.run(debug=True)
